.circleci/config.yml

```diff
@@ -1034,6 +1034,22 @@ workflows:
             name: mysql-latest
           driver: mysql
 
+      - test-driver:
+          name: be-tests-mysql-ssl-latest-ee
+          description: "(MySQL latest - connecting with SSL)"
+          requires:
+            - be-tests-ee
+          driver: mysql
+          test-selector: :only metabase.driver.mysql-test
+          extra-env: >-
+            MB_DB_TYPE=h2
+            MB_MYSQL_TEST_HOST=$MYSQL_RDS_SSL_INSTANCE_HOST
+            MB_MYSQL_TEST_SSL=true
+            MB_MYSQL_TEST_ADDITIONAL_OPTIONS='verifyServerCertificate=true'
+            MB_MYSQL_TEST_SSL_CERT="$(cat /home/circleci/metabase/metabase/resources/certificates/rds-combined-ca-bundle.pem)"
+            MB_MYSQL_TEST_USER=metabase
+            MB_MYSQL_TEST_PASSWORD=$MYSQL_RDS_SSL_INSTANCE_PASSWORD
+
```

Comment:
I think I mentioned this before, but I don't think we do a whole separate test job just for SSL. Really the only thing that's different between the two is whether we can connect or not... Can't you just pass all the connection details to one of the existing MySQL test jobs and have the connection-related tests test both the SSL and non-SSL versions?

Also, this isn't really "latest" the same way the "latest" Docker images are. It might have been "latest" whenever you spun up the RDS instance, but it probably won't be in a year.

Reply:
Something like this?  (See `mysql-connect-with-ssl-and-pem-cert-test`, which has now been added).

If this is the approach you had in mind, I will split out all of the Oracle related fixes from this PR, and move them to a separate one, and do the same thing there (adding a different test that overrides the respective Oracle env vars to test SSL connectivity in the same test).

Reply:
I took the liberty of doing that in: #15260.